src/profile/v100/entity/persona/persona_data/persona_data_identified_collection_types.rs

```diff
@@ -36,43 +36,63 @@ macro_rules! declare_collection_of_identified_entry {
         pub struct $struct_name {
             pub collection: IdentifiedVecVia<$id_ent_type>,
         }
+
         impl Default for $struct_name {
             fn default() -> Self {
-                Self::entries([])
+                Self::from_iter([])
             }
         }
+
         impl std::ops::Deref for $struct_name {
             type Target = IdentifiedVecVia<$id_ent_type>;
             fn deref(&self) -> &Self::Target {
                 &self.collection
             }
         }
-        impl $struct_name {
-            pub fn entries<I>(values: I) -> Self
+
+        impl FromIterator<$id_ent_type> for $struct_name {
```

Comment:
great to impl `FromIterator` and `IntoIterator` so that we can use `collect` and can pass something into a method accepting `IntoIterator`.